app/wama.web/Views/AdminConsole/UserTool/Index.cshtml

```diff
@@ -3,8 +3,37 @@
 
 <h2>User account summaries</h2>
 
-<h3>Accounts pending approval</h3>
-@Html.Partial("UserAccountTableDisplay")
 
-<h3>Suspended Accounts</h3>
-@Html.Partial("UserAccountTableDisplay")
+
+<center>
+    <h3>Accounts pending approval</h3>
+</center>
+<div>
+    <h4>Pending Administrators</h4>
+    @Html.Partial("UserAccountSuspendedTableDisplay", ViewData["PendingAdministrator"])
+    <h4>Pending Managers</h4>
+    @Html.Partial("UserAccountSuspendedTableDisplay", ViewData["PendingManager"])
+    <h4>Pending Employees</h4>
+    @Html.Partial("UserAccountSuspendedTableDisplay", ViewData["PendingEmployee"])
+    <h4>Pending Patrons</h4>
+    @Html.Partial("UserAccountSuspendedTableDisplay", ViewData["PendingPatron"])
+</div>
+
+
+<center>
+<h3>Suspended Account List </h3>
+    </center>
+        <div>
+            <h4>Suspended Administrators</h4>
+            @Html.Partial("UserAccountSuspendedTableDisplay", ViewData["SuspendedAdministrator"])
```

Comment:
Instead of dumping all Suspended users on one table,I am trying to categorize them to know which user is suspended. Is it Patron/Manager/Employee/Administrator?